File: src/Entity/Paiement.php
<?php

namespace App\Entity;

use App\Repository\PaiementRepository;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;

/**
 * @ORM\Entity(repositoryClass=PaiementRepository::class)
 */
class Paiement
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\Length( min = 3, max = 20, minMessage = "Merci de Vérifier Votre nom")
     * @Assert\NotBlank(message="Le champs nom sur carte est obligatoire * ")
     */
    private $NomSurCarte;

    /**
     * @ORM\Column(type="integer")
     * @Assert\Length( min=16 , max=16)
     * @Assert\NotBlank(message="Le champs NumeroTel est obligatoire * ")
     */
    private $NumCarte;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\Length( min = 3, max = 20, minMessage = "Merci de Vérifier Votre MoisExp")
     * @Assert\NotBlank(message="Le champs MoisExp est obligatoire * ")
     */
    private $MoisExp;

    /**
     * @ORM\Column(type="integer")
     * @Assert\Length( min=4 , max=4)
     * @Assert\NotBlank(message="Le champs AnneeExp est obligatoire * ")
     */
    private $AnneeExp;

    /**
     * @ORM\Column(type="integer")
     * @Assert\Length( min=3 , max=3)
     * @Assert\NotBlank(message="Le champs CVV est obligatoire * ")
     */
    private $CVV;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getNomSurCarte(): ?string
    {
        return $this->NomSurCarte;
    }

    public function setNomSurCarte(string $NomSurCarte): self
    {
        $this->NomSurCarte = $NomSurCarte;

        return $this;
    }

    public function getNumCarte(): ?int
    {
        return $this->NumCarte;
    }

    public function setNumCarte(int $NumCarte): self
    {
        $this->NumCarte = $NumCarte;

        return $this;
    }

    public function getMoisExp(): ?string
    {
        return $this->MoisExp;
    }

    public function setMoisExp(string $MoisExp): self
    {
        $this->MoisExp = $MoisExp;

        return $this;
    }

    public function getAnneeExp(): ?int
    {
        return $this->AnneeExp;
    }

    public function setAnneeExp(int $AnneeExp): self
    {
        $this->AnneeExp = $AnneeExp;

        return $this;
    }

    public function getCVV(): ?int
    {
        return $this->CVV;
    }

    public function setCVV(int $CVV): self
    {
        $this->CVV = $CVV;

        return $this;
    }
}

File: src/workshopjdbc/entities/paiement.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package workshopjdbc.entities;

import workshopjdbc.entities.Utilisateur;

/**
 *
 * @author ziedm
 */
public class paiement {
    
    private int id;

    private String nom_sur_carte;
    
    private int num_carte ;
    
    private String mois_exp;
    
    private int annee_exp;
    
    private int cvv ;
  



    public paiement() {
    }

    public paiement(int id, String nom_sur_carte, int num_carte, String mois_exp, int annee_exp, int cvv) {
        this.id = id;
        this.nom_sur_carte = nom_sur_carte;
        this.num_carte = num_carte;
        this.mois_exp = mois_exp;
        this.annee_exp = annee_exp;
        this.cvv = cvv;
    }

    public paiement(String nom_sur_carte, int num_carte, String mois_exp, int annee_exp, int cvv) {
        this.nom_sur_carte = nom_sur_carte;
        this.num_carte = num_carte;
        this.mois_exp = mois_exp;
        this.annee_exp = annee_exp;
        this.cvv = cvv;
    }



    
    

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNomSurCarte() {
        return nom_sur_carte;
    }

    public void setNomSurCarte(String nom_sur_carte) {
        this.nom_sur_carte = nom_sur_carte;
    }

    public int getNumCarte() {
        return num_carte;
    }

    public void setNumCarte(int num_carte) {
        this.num_carte = num_carte;
    }

    public String getMoisExp() {
        return mois_exp;
    }

    public void setMoisExp(String mois_exp) {
        this.mois_exp = mois_exp;
    }

    public int getAnneeExp() {
        return annee_exp;
    }

    public void setAnneeExp(int annee_exp) {
        this.annee_exp = annee_exp;
    }

    public int getCvv() {
        return cvv;
    }

    public void setCvv(int cvv) {
        this.cvv = cvv;
    }

    @Override
    public String toString() {
        return "paiement{" + "id=" + id + ", nom_sur_carte=" + nom_sur_carte + ", num_carte=" + num_carte + ", mois_exp=" + mois_exp + ", annee_exp=" + annee_exp + ", cvv=" + cvv + '}';
    }


    
    
    
}
